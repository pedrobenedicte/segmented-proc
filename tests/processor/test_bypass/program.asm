nop
add r3 <- r1, r2
add r4 <- r3, r2
nop
