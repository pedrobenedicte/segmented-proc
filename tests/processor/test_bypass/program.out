00
00
ca
40
1a
41
00
00
